src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1311,6 +1311,22 @@ public function getMimeType($format)
     }
 
     /**
+     * Gets the mime types associated with the format.
+     *
+     * @param string $format The format
+     *
+     * @return array The associated mime types
+     */
+    public function getMimeTypes($format)
```

Comment:
:+1: we should also update the other related functions in another PR IMO: [``setFormat``](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Request.php#L1343), [``getMimeType``](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Request.php#L1304) and [``getFormat``](https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Request.php#L1320).